conda_smithy/templates/azure-pipelines-linux.yml.tmpl

```diff
@@ -5,10 +5,30 @@
 jobs:
 - job: linux
   pool:
+  {%- if azure.pool is defined %}
+    # self-hosted agent
+    {%- if azure.pool.name is defined %}
+    name: {{ azure.pool.name }}
+    {%- else %}
+    name: {{ azure.pool }}
+    {%- endif %}
```

Comment:
Apparently `pool: pool_name` (the attributes `name` and `demands` are both omitted) is a valid expression.